core/src/network/reducers/respond_get_links.rs

```diff
@@ -33,19 +32,14 @@ fn reduce_respond_get_links_inner(
 }
 
 pub fn reduce_respond_get_links(
-    context: Arc<Context>,
     network_state: &mut NetworkState,
+    _root_state: &State,
     action_wrapper: &ActionWrapper,
 ) {
     let action = action_wrapper.action();
     let (get_dht_meta_data, links) = unwrap_to!(action => crate::action::Action::RespondGetLinks);
     let result = reduce_respond_get_links_inner(network_state, get_dht_meta_data, links);
 
-    context.log(format!(
-        "debug/reduce/get_links: Responding to GET LINKS request from {} with {:?}",
-        get_dht_meta_data.requester_agent_id, links
-    ));
-
```

Comment:
Hmmm we might need a way to do our logs here, it usually helps when debugging and I think we should probably have a way to log this in